.github/workflows/validation.yaml

```diff
@@ -53,7 +53,9 @@ jobs:
           SD_ENABLE_CHECK: ${{ inputs.enable-staged-dependencies-check }}
 
       - name: Build report 🏗
-        uses: insightsengineering/thevalidatoR@v1.1.1
+        uses: insightsengineering/thevalidatoR@v1.1.2
```

Comment:
Technically 1.2.0, since you added a new feature, no?

Reply:
It depends, basically tool functionality didn't change - adding new variable was related to fixing bug

Reply:
Adding a new flag/feature would induce a minor version upgrade, if we're to follow semantic versioning standards, even if it comes with a bugfix.

In the release notes, we'd include 2 items - the bugfix as well as the new flag/variable/setting.